From e34867c7b4a51628fd4e54cb9bcd5a6953bb29e0 Mon Sep 17 00:00:00 2001
From: Shadi Ammouri <shadi@marvell.com>
Date: Wed, 28 Oct 2015 13:30:44 +0200
Subject: [PATCH 107/538] version: Use Kernel auto version and localversion for
 LSP ver printout

	- auto version will add git related version to version string
	- localversion will be used to indicate the LSP version

Change-Id: I3138d8a53cc5e0f77df7f2a15a4e3c3c0f5ea7b9
Signed-off-by: Shadi Ammouri <shadi@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/24462
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Yehuda Yitschak <yehuday@marvell.com>
---
 arch/arm64/configs/mvebu_v8_lsp_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/mvebu_v8_lsp_defconfig b/arch/arm64/configs/mvebu_v8_lsp_defconfig
index 8a5f1b3..7258fc6 100644
--- a/arch/arm64/configs/mvebu_v8_lsp_defconfig
+++ b/arch/arm64/configs/mvebu_v8_lsp_defconfig
@@ -1,4 +1,3 @@
-# CONFIG_LOCALVERSION_AUTO is not set
 CONFIG_SYSVIPC=y
 CONFIG_POSIX_MQUEUE=y
 CONFIG_FHANDLE=y
-- 
1.9.1

